samples/javascript/README.md

```diff
@@ -0,0 +1,37 @@
+# Export API Javascript Example
+
+In this JavaScript example, build durations are calculated and written out to the JavaScript console.
+
+## Setup
```

Comment:
I would not call this title Setup as it is including also the running, etc. To me, ATM it seems more like the `Instructions`.